utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -691,6 +715,95 @@ int main(int argc, char **argv)
+            Rpp32u cropRegion2[4] ;
+            Rpp32u cropRegion3[4] ;
+            Rpp32u cropRegion4[4] ;
+            for (uint i = 0; i < images; i++ )
+            {
+                initialPermuteArray[i] = i;
+            }
+            randomize(initialPermuteArray, images);
+            memcpy(permutedArray, initialPermuteArray, images * sizeof(Rpp32u));
+            randomize(initialPermuteArray, images);
+            memcpy(permutedArray + images, initialPermuteArray, images * sizeof(Rpp32u));
+            randomize(initialPermuteArray, images);
+            memcpy(permutedArray + (images * 2), initialPermuteArray, images * sizeof(Rpp32u));
+            randomize(initialPermuteArray, images);
+            memcpy(permutedArray + (images * 3), initialPermuteArray, images * sizeof(Rpp32u));
+
+            RpptROI *roiPtrInputCropRegion = (RpptROI *) calloc(4, sizeof(RpptROI));
+            // xywhROI override sample
+            std::cerr<<"\n HERE 2";
+            std::cerr<<"Ix:"<<Ix;
+            std::cerr<<"Iy:"<<Iy;
+            roiPtrInputCropRegion[0].xywhROI.roiWidth = std::round(randFromDist * Ix);                    // w1
+            roiPtrInputCropRegion[0].xywhROI.roiHeight = std::round(randFromDist1 * Iy);                  // h1
+            roiPtrInputCropRegion[1].xywhROI.roiWidth = Ix - roiPtrInputCropRegion[0].xywhROI.roiWidth;   // w2
+            roiPtrInputCropRegion[1].xywhROI.roiHeight = roiPtrInputCropRegion[0].xywhROI.roiHeight;      // h2
+            roiPtrInputCropRegion[2].xywhROI.roiWidth = roiPtrInputCropRegion[0].xywhROI.roiWidth;        // w3
+            roiPtrInputCropRegion[2].xywhROI.roiHeight = Iy - roiPtrInputCropRegion[0].xywhROI.roiHeight; // h3
+            roiPtrInputCropRegion[3].xywhROI.roiWidth = Ix - roiPtrInputCropRegion[0].xywhROI.roiWidth;   // w4
+            roiPtrInputCropRegion[3].xywhROI.roiHeight = Iy - roiPtrInputCropRegion[0].xywhROI.roiHeight; // h4
+
+            roiPtrInputCropRegion[0].xywhROI.xy.x = random_val(0, Ix - roiPtrInputCropRegion[0].xywhROI.roiWidth);  // x1
+            roiPtrInputCropRegion[0].xywhROI.xy.y = random_val(0, Iy - roiPtrInputCropRegion[0].xywhROI.roiHeight); // y1
+            roiPtrInputCropRegion[1].xywhROI.xy.x = random_val(0, Ix - roiPtrInputCropRegion[1].xywhROI.roiWidth);  // x2
+            roiPtrInputCropRegion[1].xywhROI.xy.y = random_val(0, Iy - roiPtrInputCropRegion[1].xywhROI.roiHeight); // y2
+            roiPtrInputCropRegion[2].xywhROI.xy.x = random_val(0, Ix - roiPtrInputCropRegion[2].xywhROI.roiWidth);  // x3
+            roiPtrInputCropRegion[2].xywhROI.xy.y = random_val(0, Iy - roiPtrInputCropRegion[2].xywhROI.roiHeight); // y3
+            roiPtrInputCropRegion[3].xywhROI.xy.x = random_val(0, Ix - roiPtrInputCropRegion[3].xywhROI.roiWidth);  // x4
+            roiPtrInputCropRegion[3].xywhROI.xy.y = random_val(0, Iy - roiPtrInputCropRegion[3].xywhROI.roiHeight); // y4
+
+        // Change RpptRoiType for ltrbROI override sample
```

Comment:
Have you verified the ltrbROI sample too?

Reply:
Added the cases for testing ltrbROI too.